src/System.Text.Http.Parser/BadHttpRequestException.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+
+namespace System.Text.Http.Parser
+{
+    internal static class StatusCodes
+    {
```

Comment:
> @stephentoub, are you going to add these values?

We should.  Obviously I'm not going to before this PR gets merged, but we should run it though the API process and get them added.